uploadNeuroDB/imaging_upload_file.pl

```diff
@@ -337,6 +359,18 @@ =head2 Methods
             . "inserted into the database \n";
 spool($message,'N', $notify_notsummary);
 
+################################################################
+############### Email last completion message ##################
+################################################################
+$message = "\nProgress: Success\n"
+            . "$minc_created minc file(s) created, "
+            . "and $minc_inserted minc file(s) "
+            . "inserted into the database \n";
+open MAIL, "| mail $mail_user";
```

Comment:
@cmadjar do you know how I would list here either each inserted scans's minc file name or its scan_type/acquisition?

Reply:
In theory, the scan type is already in the filename so I am not sure it is necessary to add the scan type to the array here since it will list the files that were inserted.